cmd/sealer/cmd/alpha/mount.go

```diff
@@ -16,121 +16,196 @@ package alpha
+		images:     images,
+		containers: containers,
+		builders:   builders,
+	}, nil
+}
 
-			// remove destination dir if it exists, otherwise the Symlink will fail.
-			if _, err = os.Stat(path); err == nil {
-				return fmt.Errorf("destination directionay %s exists, you should remove it first", path)
+func (m Mounter) getMountedImageName(container storage.Container) string {
+	for _, image := range m.images {
+		for _, name := range image.Names {
+			if container.ImageID == image.ID {
+				return name
 			}
+		}
+	}
+	return ""
+}
 
-			mountPoint := mounts[0].MountPoint
-			if err := os.Symlink(mountPoint, path); err != nil {
-				return err
+func (m Mounter) generateMountList() error {
+	table := tablewriter.NewWriter(common.StdOut)
+	table.SetHeader([]string{imageName, tableHeaderContainerID, tableHeaderMountPath})
+	builders, err := imagebuildah.OpenBuilders(m.store)
+	if err != nil {
+		return fmt.Errorf("reading build Containers: %w", err)
+	}
+	for _, builder := range builders {
+		mounted, err := builder.Mounted()
+		if err != nil {
+			return err
+		}
+		for _, container := range m.containers {
+			if builder.ContainerID == container.ID && mounted {
+				name := m.getMountedImageName(container)
+				containerID := imagebuildah.TruncateID(builder.ContainerID, true)
+				table.Append([]string{name, containerID, builder.MountPoint})
 			}
+		}
+	}
+	table.Render()
+	return nil
+}
 
-			logrus.Infof("mount cluster image %s to %s successful", args[0], path)
-			return nil
-		},
+func (m Mounter) getMountedImageID(container storage.Container) string {
+	var imageID string
+	for _, image := range m.images {
+		if container.ImageID == image.ID {
+			imageID = image.ID
+		}
 	}
-	return mountCmd
+	return imageID
 }
 
-func mountList(images []storage.Image) error {
-	table := tablewriter.NewWriter(common.StdOut)
-	table.SetHeader([]string{imageName, "mountpath"})
-	for _, i := range images {
-		for _, name := range i.Names {
-			err := filepath.Walk(common.DefaultLayerDir, func(path string, f os.FileInfo, err error) error {
-				if f.Name() == i.ID {
-					table.Append([]string{name, filepath.Join(common.DefaultLayerDir, i.ID)})
-				}
-				return nil
+func (m Mounter) getImageID(name string) string {
+	for _, image := range m.images {
+		if strings.HasPrefix(image.ID, name) {
+			return image.ID
+		}
+		for _, n := range image.Names {
+			if name == n {
+				return image.ID
+			}
+		}
+	}
+	return ""
+}
+
+func (m Mounter) mount(imageNameOrID []string) error {
+	var imageIDList []string
+	for _, builder := range m.builders {
+		mounted, err := builder.Mounted()
+		if err != nil {
+			return err
+		}
+		for _, container := range m.containers {
+			if builder.ContainerID == container.ID && mounted {
+				imageID := m.getMountedImageID(container)
+				imageIDList = append(imageIDList, imageID)
+			}
+		}
+	}
+
+	for _, name := range imageNameOrID {
+		imageID := m.getImageID(name)
+		if ok := utilsstrings.IsInSlice(imageID, imageIDList); !ok {
```

Comment:
ok := utilsstrings.IsInSlice(imageID, imageIDList)
if ok{
logrus.Infof("this image has already been mounted, please do not repeat the operation")
}
// do others